docs/abi-spec.rst

```diff
@@ -580,15 +580,14 @@ As an example, the code
 
 ::
 
-    pragma solidity >=0.4.19 <0.7.0;
```

Comment:
What is the 0.6 feature this uses? Or is it just about the abiencoderv2 warning?

Reply:
It was a warning, and now I am ignoring warning for this case, so will revert this change